### DISCUSSION TITLE
BackgroundTasks inheritance

### DISCUSSION THREAD

**Discussion by Vert53** [encode/starlette#2633] [Potential Issue] [answered]
Hi all,
I'm wondering why does BackgroundTasks inheriting from BackgroundTask ?
BackgroundTasks doesn't use any of the attributes or methods defined in BackgroundTask. It defines its own init method and call method.

https://github.com/encode/starlette/blame/6f863b0d3b8e8f18d5df9e8cd2514f7085b874e1/starlette/background.py#L33

**Comment by adriangb** [comment, marked as answer]
It is probably not a necessary inheritance.
